Task2Maths/Task2Maths/Program.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task2Maths
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Условие Задачи 21: Дан радиус окружности найти площади вписанных в нее квадрата и правильного треугольника.\r\n");
+            Console.WriteLine("Введите радиус описанной окружности:\r\n");
+
+            double R = double.Parse(Console.ReadLine());
```

Comment:
Локальные переменные принято называть с маленькой буквы
```suggestion
            double r = double.Parse(Console.ReadLine());
```